.github/workflows/ci-tests.yml

```diff
@@ -19,9 +19,9 @@ jobs:
                     - "3.8"
                     - "3.9"
                     - "3.10"
-                    - "pypy-3.8"
+                    - "3.11"
+                    - "pypy-3.9"
```

Comment:
It would probably be best to keep `pypy-3.8` present and simply add `pypy-3.9`.

Reply:
Done 😄